astropy/units/format/vounit.py

```diff
@@ -196,7 +196,11 @@ def _format_superscript(cls, number):
         return f"({number})" if "/" in number or "." in number else f"**{number}"
 
     @classmethod
-    def to_string(cls, unit):
+    def format_exponential_notation(cls, val, format_spec=".8g"):
+        return super().format_exponential_notation(val, format_spec)
+
+    @classmethod
+    def to_string(cls, unit, inline=True):
```

Comment:
Should machine readable output formats have options like `inline` here?

Reply:
Indeed, you are right, one can never have a "non-inline" format for FITS and VOUNIT. In a way, this is kind of a placeholder for a change to `fraction` in #14449 - I think FITS at least allows in-line fractions. 

Since the same holds for `cds` where currently there is no check either (since it just inherits), how about we keep it here but ensure that in #14449 anything that is not allowed by the standard raises an error?